packages/site/src/layout/ComponentView.tsx

```diff
@@ -187,44 +186,55 @@ export const ComponentView = () => {
   };
 
   return PageMeta ? (
-    <Grid>
-      <Grid.Cell size={isMinimal ? { xs: 12, md: 12 } : { xs: 12, md: 9 }}>
```

Comment:
Was hoping you'd weigh in 😄 does this impact the VSCode integration? I assumed that if this appears in a VSCode panel that it would be less than 768, but if it's causing issues there happy to solve differently